src/hotspot/share/runtime/globals.hpp

```diff
@@ -2117,6 +2117,11 @@ const intx ObjectAlignmentInBytes = 8;
       "Throw CheckpointException if uncheckpointable resource handle found")\
                                                                             \
   product(bool, CRTrace, true, "Minimal C/R tracing")                       \
+                                                                            \
+  product(ccstr, CRIgnoredFileDescriptors, NULL, "Comma-separated list of " \
+      "file descriptor numbers or paths. All file descriptors greater than" \
```

Comment:
What if we have a file with only digits in its name, we won't be able to differentiate is that a fd or file name, right?

Reply:
That's right; in such case it is a relative file and you can specify it as `./foobar`.